locale/locales/en.json

```diff
@@ -31,6 +31,34 @@
     "value": "Last page",
     "comment": "tooltip and aria-label for the last page button. (tew 210929)"
   },
+  "SNTable.Pagination.MovedToFirstPage": {
+    "value": "You are in the first page.",
```

Comment:
```suggestion
    "value": "You are on the first page.",
```

Reply:
and same for the rest

Reply:
Actually these should be the same as `MovedToNextPage`
```suggestion
    "value": "Moved to the first page.",
```

Reply:
✅ fixed